### ISSUE TITLE
Sybil Attacker Report

### ISSUE THREAD

**Issue by shdwy** [hop-protocol/hop-airdrop#354]
# Related Addresses

```
Part 1
0x6d63034b0b31c88c3667a4f35ca4793ccab73bfd
0x9bea6364d6465dbf7a8efd5ca5699881b2d5f525
0x483b7fc91dc1b4a2361fb98eccb8034f1ff9a914
0x4b710f9da3ff4a85284198aad2270775e2544f78
0x7799648908de1565a9496098cbeb0257cf069b8c
0x176396839fa923ec224e391da85089a16323e350
0x276cd992ef150f7ab40caedea8eca429ec80b1ba
0x450af8222eb06e87b3566c7ff2e0e2a425427699
0xb7a33f067799290353e225120c17b219d81c545d
0xc27771fef68b052790bd3ab4f05ae2454b0a34a9
0xa960527dd4a127afac13a8bb13ee168b379bd732
0xd3f7b3fd5903ec6567337f8ac4745160ee6a2375
0x01d36303366ee9e94c3cfed0c9dd41857d6f644d
0xc3be3e7d9cad516305b195a70495d9742be57fce
0x7615e3d68b4ab83252d649e9762cc32d1aaaebe6
0x3b10438e505d83ac1a9884e6ef41e1c32d969d09
0x90b85134578ef9e0f3427d0c429b15a9361a56af
0x7fd9b6e2e4296a13577e6f8ab312da43df706e77 

Part 2
0x6f35109ddc4f3d1111b2ed82273b07755a1e923a
0x000c1aede0824a53f5ba7d6c8cc66f4b4b631c1a
0x34531afc6d2e077e2faca6e57b187649bbec1d0b
0x52eaaf5dec491a80f21a512e5ebabe81ad658745
0x834fc93d5fc75628981d21d0bc74597d172c3580
0xb25cad11c423b528e468d89312b29ff02de04bfb
0xa5485310c6ef4940c9e9e670557df867393e09bf
0xb6321c5e3d0a97b8d4218bc15ec95bd8d1f7d639
0x498ea383d5736635afcf72caddcc86b809e06cdb
0x3ce8500c5b0898c96a00287154794b4cf1fce0fc
0x471deb36b700b6ebab3da6573eedc2db8ed1af89
0x5e1d4b9eba99fcbe8cce969f52ad5d8a2db31df5
0xafac76d3dd6b3744fea17975792ccb7993335352
0x74bf239a4f8abe85d3f9382b22e008ae50229ff5
0x083397bf566d24d6aa2704531ef33e7c96df9068
0x5fe933af37085ccc536a97799d6d6e83c11db76f
0x73f46d7aec489b47f7fff2233c53e82184d39e7f
0x3ae06234f9b31f37d3ec062d24ca40b7a8db9024
0xe86eee44dd5ae1e03a082fb9acdebdaf31d58783
0xbf3dc018ddf1928d40a58e18345c8a1021acbbd6



```

# Reasoning

The attack is divided on two parts:
The first was conducted on Jun-12-2021 and started on wallet 0x6d63034b0b31c88c3667a4f35ca4793ccab73bfd. 
He interacts with the same set of contracts on the same day on all addresses (mostly gaming other airdrops, like Instadapp), then moves his remaining funds to the next wallet and does mostly the same.

The seconds part was conducted on Aug-06-2021 and started on wallet 0x6f35109ddc4f3d1111b2ed82273b07755a1e923a, he interacts with the same set of contracts on the same day on all addresses and then moves his remaining funds to the next wallet to do the same.

The link between the two sets is: 
[0x417814284f7446e503cabbc204d8283319bd7e9dba7f131d6e76d55a72117987](https://polygonscan.com/tx/0x417814284f7446e503cabbc204d8283319bd7e9dba7f131d6e76d55a72117987)

--

# Methodology

I used Alchemy's API to see which address originally funded all 30k eligible addresses on Polygon Mainnet, then used excel to check if the funder's wallet was also eligible for the airdrop to try and find chains of linked wallets, I then manually checked each address history on polygonscan to corroborate my findings.

I can submit my excel workbook if need be.

# Rewards Address

0xD47CB1835EB36e4A108C33E446449192426Fc160

**Comment by shanefontaine** [comment]
Thank you for your report @shdwy. We have verified that the addresses in this report are Sybil attackers.

The report included 27 eligible addresses as Sybil attackers which means you are eligible for 6682.762070422713725394 HOP! When Hop DAO is live, we will make a proposal for this reward — subject to a 1 year lockup, as mentioned in the [original Mirror post](https://hop.mirror.xyz/AI5fOUR0X_l0mktShDOx3mwr-hsB24gp8GvTWtS-MBc).

The address `0xb6321c5e3d0a97b8d4218bc15ec95bd8d1f7d639` was not included as they have different behaviors than the other addresses and there is a non-negligible probability of that being a legitimate user.

The qualified addresses are as follows:

```
0x6d63034b0b31c88c3667a4f35ca4793ccab73bfd
0x9bea6364d6465dbf7a8efd5ca5699881b2d5f525
0x483b7fc91dc1b4a2361fb98eccb8034f1ff9a914
0x4b710f9da3ff4a85284198aad2270775e2544f78
0x7799648908de1565a9496098cbeb0257cf069b8c
0x176396839fa923ec224e391da85089a16323e350
0x276cd992ef150f7ab40caedea8eca429ec80b1ba
0x450af8222eb06e87b3566c7ff2e0e2a425427699
0xb7a33f067799290353e225120c17b219d81c545d
0xc27771fef68b052790bd3ab4f05ae2454b0a34a9
0xa960527dd4a127afac13a8bb13ee168b379bd732
0xd3f7b3fd5903ec6567337f8ac4745160ee6a2375
0x01d36303366ee9e94c3cfed0c9dd41857d6f644d
0xc3be3e7d9cad516305b195a70495d9742be57fce
0x7615e3d68b4ab83252d649e9762cc32d1aaaebe6
0x3b10438e505d83ac1a9884e6ef41e1c32d969d09
0x90b85134578ef9e0f3427d0c429b15a9361a56af
0x7fd9b6e2e4296a13577e6f8ab312da43df706e77
0x000c1aede0824a53f5ba7d6c8cc66f4b4b631c1a
0x34531afc6d2e077e2faca6e57b187649bbec1d0b
0x52eaaf5dec491a80f21a512e5ebabe81ad658745
0x834fc93d5fc75628981d21d0bc74597d172c3580
0xb25cad11c423b528e468d89312b29ff02de04bfb
0xa5485310c6ef4940c9e9e670557df867393e09bf
0x498ea383d5736635afcf72caddcc86b809e06cdb
0x3ce8500c5b0898c96a00287154794b4cf1fce0fc
0xe86eee44dd5ae1e03a082fb9acdebdaf31d58783
```

**Comment by shanefontaine** [comment]
It looks like all addresses had similar behaviors on the Hop protocol as well:

address | ensName | total | mainnet | arbitrum | optimism | polygon | xdai | totalVolume
-- | -- | -- | -- | -- | -- | -- | -- | --
0x498ea383d5736635afcf72caddcc86b809e06cdb |   | 16 | 0 | 0 | 0 | 10 | 6 | $6,859.24
0xa5485310c6ef4940c9e9e670557df867393e09bf |   | 24 | 0 | 0 | 0 | 11 | 13 | $6,403.03
0x3ce8500c5b0898c96a00287154794b4cf1fce0fc |   | 16 | 0 | 0 | 0 | 10 | 6 | $5,704.65
0x6d63034b0b31c88c3667a4f35ca4793ccab73bfd |   | 21 | 0 | 1 | 0 | 13 | 7 | $5,268.48
0x9bea6364d6465dbf7a8efd5ca5699881b2d5f525 |   | 18 | 0 | 1 | 0 | 13 | 4 | $4,791.38
0xe86eee44dd5ae1e03a082fb9acdebdaf31d58783 |   | 12 | 0 | 0 | 0 | 11 | 1 | $3,694.64
0x483b7fc91dc1b4a2361fb98eccb8034f1ff9a914 |   | 18 | 0 | 0 | 0 | 3 | 15 | $3,672.65
0xd3f7b3fd5903ec6567337f8ac4745160ee6a2375 |   | 15 | 0 | 0 | 0 | 12 | 3 | $3,486.40
0x7799648908de1565a9496098cbeb0257cf069b8c |   | 16 | 0 | 0 | 0 | 3 | 13 | $3,166.87
0x4b710f9da3ff4a85284198aad2270775e2544f78 |   | 15 | 0 | 0 | 0 | 2 | 13 | $2,940.19
0xa960527dd4a127afac13a8bb13ee168b379bd732 |   | 15 | 0 | 0 | 0 | 12 | 3 | $2,938.28
0x176396839fa923ec224e391da85089a16323e350 |   | 15 | 0 | 0 | 0 | 2 | 13 | $2,872.53
0x450af8222eb06e87b3566c7ff2e0e2a425427699 |   | 28 | 0 | 0 | 0 | 24 | 4 | $2,818.22
0xb7a33f067799290353e225120c17b219d81c545d |   | 12 | 0 | 0 | 0 | 1 | 11 | $2,689.68
0xc3be3e7d9cad516305b195a70495d9742be57fce |   | 15 | 0 | 0 | 0 | 13 | 2 | $2,685.47
0x276cd992ef150f7ab40caedea8eca429ec80b1ba |   | 14 | 0 | 0 | 0 | 3 | 11 | $2,675.05
0x7615e3d68b4ab83252d649e9762cc32d1aaaebe6 |   | 13 | 0 | 0 | 0 | 11 | 2 | $2,652.20
0x3b10438e505d83ac1a9884e6ef41e1c32d969d09 |   | 11 | 0 | 0 | 0 | 9 | 2 | $2,386.31
0x01d36303366ee9e94c3cfed0c9dd41857d6f644d |   | 17 | 0 | 0 | 0 | 14 | 3 | $2,327.73
0x52eaaf5dec491a80f21a512e5ebabe81ad658745 |   | 16 | 0 | 0 | 0 | 10 | 6 | $2,220.82
0xc27771fef68b052790bd3ab4f05ae2454b0a34a9 |   | 14 | 0 | 0 | 0 | 11 | 3 | $2,084.82
0x7fd9b6e2e4296a13577e6f8ab312da43df706e77 |   | 20 | 0 | 0 | 0 | 17 | 3 | $2,077.46
0xb25cad11c423b528e468d89312b29ff02de04bfb |   | 17 | 0 | 0 | 0 | 10 | 7 | $1,961.62
0x34531afc6d2e077e2faca6e57b187649bbec1d0b |   | 17 | 0 | 0 | 0 | 10 | 7 | $1,875.55
0x834fc93d5fc75628981d21d0bc74597d172c3580 |   | 14 | 0 | 0 | 0 | 12 | 2 | $1,848.89
0x90b85134578ef9e0f3427d0c429b15a9361a56af |   | 13 | 0 | 0 | 0 | 11 | 2 | $1,525.66
0x000c1aede0824a53f5ba7d6c8cc66f4b4b631c1a |   | 15 | 0 | 0 | 0 | 10 | 5 | $1,279.31

**Comment by forweb3** [comment]
I don't see any relevant reasoning process? I'm curious how this was judged? Because the whistleblower did not list the process.

**Comment by forweb3** [comment]
I don't see any relevant reasoning process? I'm curious how this was judged? Because the whistleblower didn't list the process only that he found the address.

**Comment by forweb3** [comment]
The reasoning process I see only provides a record of transfers between two addresses. Rather than a more complete reasoning process like other entries. Just relying on this as a judgment I think there will be real users who will be misjudged.

**Comment by shanefontaine** [comment]
@forweb3  All addresses in the final list exist in transaction chains on Fantom. The one that did not, `0xb6321c5e3d0a97b8d4218bc15ec95bd8d1f7d639`, was excluded from the final list because of this reason.

**Comment by forweb3** [comment]
> @forweb3 All addresses in the final list exist in transaction chains on Fantom. The one that did not, `0xb6321c5e3d0a97b8d4218bc15ec95bd8d1f7d639`, was excluded from the final list because of this reason.

okay，i see

**Comment by forweb3** [comment]
sir ，One of my, our report is finding that the rule has to be a jump to survive. However, the successor has completed more than 10 user tracking records, and there is no record of him. I cancelled his reward. And this is what I found so am I not rewarded accordingly?  This address has not interacted with hop. And he did not provide other information that he interacted. I report all his reports.
0xD47CB1835EB36e4A108C33E446449192426Fc160

**Comment by forweb3** [comment]
sir ，One of my, our report is finding that the rule has to be a jump to survive. However, the successor has completed more than 10 user tracking records, and there is no record of him. I cancelled his reward. And this is what I found so am I not rewarded accordingly? This address has not interacted with hop. And he did not provide other information that he interacted. I report all his reports.
0xD47CB1835EB36e4A108C33E446449192426Fc160 @shanefontaine
